### PR TITLE
Changelog 2.2.4

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.2.4 (Apr 4, 2015)
+### general
+ - Hide sensitive data from from the log when shutting down a stale Logstash ([#4952](https://github.com/elastic/logstash/pull/4952)).
+ - Do not output the Configuration AST by default when running Logstash in debug mode, introduce `--debug-config` flag to display the AST ([#4965](https://github.com/elastic/logstash/issues/4964)).
+ - Fix the stop command of the Ubuntu init script ([#4940](https://github.com/elastic/logstash/issues/4940)
```
Comment (colinsurprenant) [inline CHANGELOG.md]: we use the past tense: Fixed

**Comment by colinsurprenant** [comment]
Other than my inline comment on the wording consistency, LGTM

**Comment by ph** [comment]
merged https://github.com/elastic/logstash/commit/697ff373b3d9e882cc940f7649d4b3b832ff9cc5

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## 2.2.4 (Apr 4, 2015)
+### general
+ - Hided sensitive data from from the log when shutting down a stale Logstash ([#4952](https://github.com/elastic/logstash/pull/4952)).
```
Comment (colinsurprenant) [inline CHANGELOG.md]: Hid or hidden? :P

**Pull request closed without merging**